Source/DiscoveryV1/Models/TrainingExamplePatch.swift

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright IBM Corporation 2018
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/** TrainingExamplePatch. */
+public struct TrainingExamplePatch: Encodable {
```

Comment:
This model -- and maybe others -- are used internally by the SDK but not truly a "public" interface.  I wonder if we should declare such models as "internal" rather than "public"?

Reply:
Good idea. This seems to happen when:
1. The model is only used for requests not responses.
1. The model is only ever used as a body model.
1. The model is always exploded.

I searched for `// create body` in `Discovery.swift` to find models that were exploded, then set them `internal`, and fixed up any build errors for models that were referenced elsewhere.

As a side note, I started to see if I could add an access control tag to models in the generator. But I don't think we have the information we need to calculate the tag yet. (Not least of which because exploding happens in post-processing.) But here's the code in case it's helpful.

```java
// Internal annotation added to a model to specify its access level
// (i.e. open, public, internal, fileprivate, or private)
private static final String X_ACCESS_LEVEL = "x-access-level"; // String

// set access level (internal if exploded and not response model, otherwise public)
if (watsonModel.vendorExtensions == null) watsonModel.vendorExtensions = new HashMap<>();
watsonModel.vendorExtensions.put(X_ACCESS_LEVEL, "public");
if (watsonModel.isRequestModel && !watsonModel.isResponseModel && alwaysExploded) { // <-- compute alwaysExploded
    watsonModel.vendorExtensions.put(X_ACCESS_LEVEL, "internal");
}
```